public/index.html

```diff
@@ -11,30 +11,30 @@
     />
     <meta
       name="msapplication-square310x310logo"
-      content="%PUBLIC_URL%/img/icons/mstile-310x310.png"
```

Comment:
As I've mentioned already I don't think this is a good fix for the issue. 
Consider using `<%= BASE_URL %>` instead and checks vue cli docs.

`BASE_URL` help to host your site everywhere.

Let's say you will create your own site like `https://liubich.com` and you'd like to host weather app as `https://liubich.com/weather`. With current fix it won't work

Reply:
https://cli.vuejs.org/guide/html-and-static-assets.html#the-public-folder